services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configuration.entity.ts

```diff
@@ -1,63 +1,56 @@
-import { isObject } from 'lodash';
 import {
   Column,
   Entity,
   JoinColumn,
   ManyToOne,
+  OneToMany,
   Relation,
   Unique,
 } from 'typeorm';
 
 import { CascadeDeleteEntity } from '@121-service/src/base.entity';
-import { FinancialServiceProviderEntity } from '@121-service/src/financial-service-providers/financial-service-provider.entity';
+import { FinancialServiceProviderName } from '@121-service/src/financial-service-providers/enum/financial-service-provider-name.enum';
+import { TransactionEntity } from '@121-service/src/payments/transactions/transaction.entity';
+import { ProgramFinancialServiceProviderConfigurationPropertyEntity } from '@121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configuration-property.entity';
 import { ProgramEntity } from '@121-service/src/programs/program.entity';
+import { LocalizedString } from '@121-service/src/shared/types/localized-string.type';
 
-@Unique('programFspConfigurationUnique', ['programId', 'fspId', 'name'])
-@Entity('program_fsp_configuration')
+@Unique('programFinancialServiceProviderConfigurationUnique', [
+  'programId',
+  'name',
+])
+@Entity('program_financial_service_provider_configuration')
 export class ProgramFinancialServiceProviderConfigurationEntity extends CascadeDeleteEntity {
```

Comment:
We do not have to create that link from the ProgramFinancialServiceProviderConfigurationEntity since the fk is stored in the registration. This is only needed if we ever want to do a:

`this.ProgramFinancialServiceProviderConfigurationRepository.find(id { relations: ['registrations'])`

Which we do not do anywhere atm. We could still add is since we have a 2 way link in most other places (except for example in fsp specific entities to registrations or transactions). What do you think?

Reply:
Fine for now.